backend/src/index.ts

```diff
@@ -9,6 +9,7 @@ const HOST = "http://localhost:5000";
 const NAMESPACE = "default";
 const SIMPLE_STORAGE_ADDRESS = "ContractAddressHere";
 const TOKEN_ADDRESS = "ContractAddressHere";
+const SIGNING_KEY = "SigningKeyHere";
```

Comment:
We seem to have moved from the previous approach a few versions ago of:
`config.example.json` with comments, to this `index.ts` approach.

I think the previous was more obvious.
If you skipped it, the runtime complains "Erm... where is config.json at?" rather than looking like it started, but actually being broken.

Reply:
Added a default `config.json` file, with a section in the README.